services/reproducible-central/reproducible-central.service.js

```diff
@@ -0,0 +1,91 @@
+import Joi from 'joi'
+import { BaseJsonService, pathParams } from '../index.js'
+
+const schema = Joi.object({
+  color: Joi.string().required(),
```

Comment:
On the colours, normally we'd implement our own palette with semantic colours consistent across badges. This is particularly the case for things like a version number or build status which is common across lots of different services. That is less important here because I don't think this is really a common metric so whatever we end up with here is going to be kind of bespoke. However, we should at least match the normal convention that the "best" score is brightgreen (rather than green). I would be in favour of doing that by parsing out the numbers and defining the colour scheme here instead of completely ceding this to the upstream.

Reply:
ok, I'll do that as part of my change of approach cited in another comment